WordPress/Sniffs/DB/PreparedSQLPlaceholdersSniff.php

```diff
@@ -119,7 +119,7 @@ public function process_token( $stackPtr ) {
 					foreach ( $matches[0] as $match ) {
 						if ( '%' === $match ) {
 							$this->phpcsFile->addError(
-								'Found unescaped literal "%" character.',
+								'Found unescaped literal "%%" character.',
```

Comment:
This doesn't make sense now - two characters (both `%`), yet the final word of the message suggests a single character.

Reply:
That's the whole point. Both the `addError()/addWarning()` methods as well as `$wpdb->prepare()` use a `sprintf()` like function to parse in variables into the error message/SQL query.
The `%` is used as as placeholder indicator. To use a literal `%`, you need to escape it by adding a second `%`.

As in -without this fix, the error message displayed became:
`Found unescaped literal " character.`
With the fix:
`Found unescaped literal "%" character.`